task/net/src/server.rs

```diff
@@ -20,9 +20,9 @@ use task_net_api::{
 
 use core::iter::zip;
 use heapless::Vec;
-use smoltcp::iface::{Interface, Neighbor, SocketHandle, SocketStorage};
-use smoltcp::socket::UdpSocket;
-use smoltcp::wire::{EthernetAddress, IpAddress, IpCidr, Ipv6Cidr};
+use smoltcp::iface::{Interface, SocketHandle, SocketStorage};
+use smoltcp::socket::udp::Socket as UdpSocket;
```

Comment:
(slight preference for importing `udp` and calling this type `udp::Socket` so it doesn't look like a new type)

Reply:
yup, seems wise